addons/crm/models/crm_lead.py

```diff
@@ -2022,10 +2026,11 @@ def _pls_get_lead_pls_values(self, domain=[]):
                     elif field in ('email_state', 'phone_state'):  # As ORM reads 'None' as 'False', do the same here
                         lead_values.append((field, False))
                     leads_values_dict[lead['id']] = {'values': lead_values, 'team_id': lead['team_id'] or 0}
-
-            for tag in tag_results:
-                if tag['tag_id']:
-                    leads_values_dict[tag['lead_id']]['values'].append(('tag_id', tag['tag_id']))
+            if use_tags:
```

Comment:
Why not setting tag_results to [] if we don't use tags, and doing directly for tag in tag_results ? It will lessen diff.

Reply:
good eye dee